website/content/docs/concepts/seal.mdx

```diff
@@ -337,10 +335,8 @@ migration will not happen again on the peer nodes.
 
 ## Seal High Availability (Enterprise)
 
-@include 'alerts/beta.mdx'
-
-Seal High Availability (Seal HA) allows the configuration of more than one auto
-seal mechanism such that Vault can tolerate the temporary loss of a seal service
+Seal high availability (Seal HA) allows the configuration of more than one auto 
+seal mechanism such that Vault can tolerate the temporary loss of a seal service 
```

Comment:
```suggestion
Seal high availability (seal HA) lets you configure multiple auto seal
mechanisms so your Vault cluster can tolerate the temporary loss of a seal
service.
```
Style correction: write in active voice

Reply:
We use this style pretty pervasively, I'd rather tackle that kind of change more holistically so as not to have a mismatch in the docs.